core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/components/dot-category-field-dialog/dot-category-field-dialog.component.ts

```diff
@@ -26,60 +24,48 @@ import { DotCategoryFieldSearchListComponent } from '../dot-category-field-searc
 import { DotCategoryFieldSelectedComponent } from '../dot-category-field-selected/dot-category-field-selected.component';
 
 /**
- * The DotCategoryFieldSidebarComponent is a sidebar panel that allows editing of content category field.
+ * The DotCategoryFieldDialogComponent is a dialog panel that allows editing of content category field.
  * It provides interfaces for item selection and click handling, and communicates with a store
  * to fetch and update the categories' data.
  *
- * @property {boolean} visible - Indicates the visibility of the sidebar. Default is `true`.
- * @property {EventEmitter<void>} closedSidebar - Event emitted when the sidebar is closed.
+ * @property {boolean} visible - Indicates the visibility of the dialog. Default is `true`.
+ * @property {EventEmitter<void>} closedDialog - Event emitted when the dialog is closed.
  */
 @Component({
-    selector: 'dot-category-field-sidebar',
+    selector: 'dot-category-field-dialog',
     standalone: true,
     imports: [
         DialogModule,
         ButtonModule,
         DotMessagePipe,
-        SidebarModule,
         DotCategoryFieldCategoryListComponent,
         InputTextModule,
         DotCategoryFieldSearchComponent,
         DotCategoryFieldSearchListComponent,
         DotCategoryFieldSelectedComponent,
         NgClass
     ],
-    templateUrl: './dot-category-field-sidebar.component.html',
-    styleUrl: './dot-category-field-sidebar.component.scss',
-    changeDetection: ChangeDetectionStrategy.OnPush,
-    animations: [
-        trigger('fadeAnimation', [
-            state(
-                'void',
-                style({
-                    opacity: 0
-                })
-            ),
-            transition(':enter, :leave', [animate('50ms ease-in-out')])
-        ])
-    ]
+    templateUrl: './dot-category-field-dialog.component.html',
+    styleUrl: './dot-category-field-dialog.component.scss',
+    changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class DotCategoryFieldSidebarComponent implements OnInit, OnDestroy {
+export class DotCategoryFieldDialogComponent implements OnInit, OnDestroy {
     /**
-     * Indicates the visibility of the sidebar.
+     * Indicates the visibility of the dialog.
      *
-     * @memberof DotCategoryFieldSidebarComponent
+     * @memberof DotCategoryFieldDialogComponent
      */
     @Input() visible = false;
 
     /**
-     * Output that emit if the sidebar is closed
+     * Output that emit if the Dialog is closed
      */
-    @Output() closedSidebar = new EventEmitter<void>();
+    @Output() closedDialog = new EventEmitter<void>();
```

Comment:
@hmoreras I suggest using the new API for outputs and transforms this into: `closedDialog = output<void>();`